src/lib/tooltip/tooltip.ts

```diff
@@ -217,12 +217,16 @@ export class MatTooltip implements OnDestroy {
         .set('mouseenter', () => this.show())
         .set('mouseleave', () => this.hide())
         .forEach((listener, event) => element.addEventListener(event, listener));
-    } else if (_platform.IOS && (element.nodeName === 'INPUT' || element.nodeName === 'TEXTAREA')) {
+    }
+
+    if ((_platform.IOS || _platform.SAFARI) &&
```

Comment:
At this point, I think that we should do this on all browsers. Text selection is something pretty fundamental to inputs.

Reply:
I agree. Also the the `user-select: none` is mostly needed if something is being panned/dragged. So we should be good always removing the `user-select`.

Reply:
Done. I've updated the commit message, but kept the title because the changelog should show that the specific safari case has been fixed.